packages/vapor/resources/icons/svg/unlock-stroked-large.svg

```diff
@@ -0,0 +1,5 @@
+<svg width="32" height="32" viewBox="0 0 32 32" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M4.5 17C4.5 16.1716 5.17157 15.5 6 15.5H18C18.8284 15.5 19.5 16.1716 19.5 17V26C19.5 26.8284 18.8284 27.5 18 27.5H6C5.17157 27.5 4.5 26.8284 4.5 26V17Z" stroke="#282829"/>
```

Comment:
For stroked icons, we set `stroke="#282829"` as the default color on the root of `svg`, on the line 1 of `.svg` file. And then we can change color globally with classes, for example,`.icon.mod-stroke.mod-warning`.